Zendesk v2 permissions documentations (#24420)
demisto/content: Packs/Zendesk/Integrations/Zendeskv2/README.md

    | Use system proxy settings |  | False |
    | Trust any certificate (not secure) |  | False |

4. Click **Test** to validate the URLs, token, and connection. 

>     (The test does not ensure sufficient permissions for all integration commands.)

Reviewer: Don't we want to test permissions for all the roles and notify the user what fails rather than let him guess ?

Author: He does not need to guess I added documentation for each command - what permissions it requires.

## Commands
You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
After you successfully execute a command, a DBot message appears in the War Room with the command details.

## Required permissions

> This integration enables executing commands with different permission levels. plese look at the commands description for more information on the required permission. 

Reviewer 2: ```suggestion
This integration enables executing commands with different permission levels. See the commands' descriptions for more information on the required permission. 
```

### zendesk-user-list
***
Gets the specified user's data. 

> Required permmisions: Admins, Agents and Light Agents.

Reviewer 2: ```suggestion
Required permissions: Admins, Agents and Light Agents.
```

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |
| user_id | The user's ID to retrieve. | Optional | 

> | user_name | The user's name. <br/>Required permmisions to use this argument: Agents . | Optional | 

Reviewer 2: ```suggestion
| user_name | The user's name. <br/>Required permissions to use this argument: Agents . | Optional | 
```

> | user_name | The user's name. <br/>Required permissions to use this argument: Agents . | Optional | 

Reviewer: Why do you add the `<br/>` ?  It's an HTML tag

Author: Its auto-generated by the SDK generate-docs i guess it identifies the new breakline from the yml and pasted it here.

Author: ![image](https://user-images.githubusercontent.com/95565843/217271420-382b788e-d00d-444f-9a32-985169a28d89.png)

### zendesk-user-create
***
Creates a new Zendesk user. 

> Required permissions: Agents, with restrictions on certain actions

Reviewer 2: ```suggestion
Required permissions: Agents, with restrictions on certain actions.
```

### zendesk-user-update
***
Update user data. 

> Required permissions: Agents, with restrictions on certain actions

Reviewer 2: ```suggestion
Required permissions: Agents, with restrictions on certain actions.
```

### zendesk-user-delete
***
Delete a user. 

> Required permissions: Admins 

Reviewer 2: ```suggestion
Required permissions: Admins.
```

There is no context output for this command.
### zendesk-organization-list
***
Get organization's data. 
Required permissions: Agents, with certain restrictions. 

> If the agent has a custom agent role that restricts their access to only users in their own organization,

Reviewer 2: ```suggestion
If the agent has a custom agent role that restricts the agents access to only users in their own organization,
```

> a 403 Forbidden error is returned. See Creating custom agent roles in Zendesk help.

Reviewer 2: ```suggestion
a 403 Forbidden error is returned. See Creating custom agent roles in the Zendesk help.
```

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |

> | organization_id | The ID of a specific organization. <br/>Required permissions: Admins, Agents.  | Optional | 

Reviewer: Why do you add the `<br/>` ?  It's an HTML tag

Author: same

> | organization_id | The ID of a specific organization. <br/>Required permissions: Admins, Agents. . | Optional | 

Reviewer 2: ```suggestion
| organization_id | The ID of a specific organization. <br/>Required permissions: Admins, Agents.  | Optional | 
```

### zendesk-ticket-create
***
Create a new zendesk ticket.

> Required permissions: Agents

Reviewer 2: ```suggestion
Required permissions: Agents.
```

### zendesk-ticket-update
***
Updates a Zendesk ticket.

> Required permissions: Agents 

Reviewer 2: ```suggestion
Required permissions: Agents.
```

### zendesk-ticket-delete
***
Delete ticket. 
Required permissions: Admins, Agents with permission to delete tickets.
Agent delete permissions are set in Support.

> See Deleting tickets in the Zendesk Support Help Cent

Reviewer 2: ```suggestion
See Deleting tickets in the Zendesk Support Help Center.
```

### zendesk-ticket-comment-list
***
List comments for a given ticket.

> Required: Agents

Reviewer 2: ```suggestion
Required: Agents.
```

> Required: Agents.

Reviewer: ```suggestion
Required permissions: Agents.
```

### zendesk-ticket-attachment-add
***
Attach file to ticket.

> Required permissions: End users

Reviewer 2: ```suggestion
Required permissions: End users.
```

### zendesk-article-list
***
List all available articles.

> Rquired permissions: Agents, End users, Anonymous users.

Reviewer 2: ```suggestion
Required permissions: Agents, End users, Anonymous users.
```
